LocalSettings.php

```diff
@@ -5545,6 +5545,26 @@
 		'wisdomwikiwiki' => 'freenodeChat',
 		'wisdomsandboxwiki' => 'freenodeChat',
 	),
+	
+	//WhoIsWatching
+	$whoiswatching_nametype => array(
```

Comment:
Please do it in the format that all the other variables are done which requires a ', for example '$wgWhoIsWatching_Nametype', etc.

Also the "= array" is supposed to be a "=> array"